wiki/en/en.FAQ.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: wiki
+title: "FAQ"
+lang: "en"
+permalink: "/wiki/FAQ"
+---
+
+# Jamulus FAQ
+
+This is an overview of common questions. For common problems and their solutions, see the [Troubleshooting](/wiki/Client-Troubleshooting) page.
+
+
+### Is there a metronome, synchronization, or some other way of keeping in time?
+
+No. Musicians on a Jamulus server simply play in real time time together as they would in person. If you want to have a time signal, then there are shared metronome solutions on the Internet you can try, but it’s probably best to just minimise latency so nobody has an overall delay more than about 30-50ms. 
+
+### How do I know if I can join a server? Are there rules?
+
+In general, if somebody sets up a public server, then [by definition](/wiki/Choosing-a-Server-Type) they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms. However, some servers may state their policies in the welcome message you will see in the chat window.
+
+You can also set up a private server on Jamulus and give others your address to connect to that. Have a look at [this page for more information](/wiki/Running-a-Server).
+
+### Why shouldn’t I use wireless equipment?
+
+Whether or not you will be able to play in time with other musicians depends mainly on how much latency you have in your sound signal. An overall delay much bigger than 50ms usually makes it too hard to play in time unless you train yourself to do so. Some musicians report they can play in time with delays of up to 70ms by playing ahead of the beat. But much beyond that and it becomes too difficult unless timing isn’t an important factor in the music.
+
+So it makes sense to minimise any sources of delay or other problems with the signal. Even fast wi-fi is usually too variable to use for long periods with Jamulus, and Bluetooth headphones and mics usually introduce latencies of about 50ms or more. 
+
+### Why should I not listen to my own signal?
+
+For the same reason as you need to minimise delay in your signal in order to play in time, you need to make sure you are playing to your own sound that other musicians are hearing. More information on this, and a way of testing your setup to make sure you are obeying this “Golden Rule” [can be found here](/wiki/Client-Troubleshooting).
+
+Of course, if you are playing an acoustic instrument, or are a singer, it will be hard to exclude your “local” sound. But you should at least try to do so by for example using closed back headphones turned up as loud as you can to mask your own sound so that you can concentrate on the mix coming back to you from the server.
```

Comment:
> turned up as loud as you can

You'll have the Health And Safety Department on to you.  "as best you can" might be more acceptable :)

Reply:
"Turn it up until you hear nothing else"? "As much as you can"? "To eleven"? "Until unconsciousness or death intervene"?